advanced/am-pattern-3/templates/am/tm/instance-2/wso2am-pattern-3-am-tm-2-deployment.yaml

```diff
@@ -119,6 +95,9 @@ spec:
         - name: wso2am-pattern-3-am-tm-2-conf
           mountPath: /home/wso2carbon/wso2-config-volume/repository/conf/deployment.toml
           subPath: deployment.toml
+        - name: wso2am-pattern-3-conf-entrypoint
```

Comment:
@Chathuru why do we need to override the entry point here?

Reply:
Fix https://github.com/wso2/kubernetes-apim/pull/365/commits/3588e9f5044c3947741ef019ecda0ee150daad18